release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -248,41 +249,63 @@ class CreateReleaseSubstepsState extends State<CreateReleaseSubsteps> {
   }
 }
 
-typedef SetReleaseData = void Function(String name, String data);
-typedef ChangeIsEachInputValid = void Function(String name, bool isValid);
+typedef SetReleaseData = void Function(CreateReleaseSubstep name, String data);
+typedef ChangeIsEachInputValid = void Function(CreateReleaseSubstep name, bool isValid);
 
-/// Captures the input values and updates the corresponding field in [releaseData].
+/// Captures and validates the input values and updates the corresponding field in [releaseData].
+///
+/// [substep] parameter is a [CreateReleaseSubstep] enum that represents the current substep
```

Comment:
You don't need to include `parameter` after square brackets in dartdocs (its repetitive)

Reply:
Same about `enum` as any IDE will indicate CreateReleaseSubstep is an enum